.github/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Anton's portfolio website</title>
+    <link rel="stylesheet" href="style.css">
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+</head>
+<body>
+    <header class="header">
+        <a href="#" class="logo">Anton</a>
+        <nav class="navbar">
+            <a href="#" class="active">Home</a>
+            <a href="#">About</a>
+            <a href="#">Experience</a>
+            <a href="#">Portfolio</a>
+        </nav>
+    </header>
+
+    <section class="home">
+        <div class="home-content">
```

Comment:
consider semantic elements instead of some of these divs, to make the page more accessible - https://www.w3schools.com/html/html5_semantic_elements.asp